git_workflow (updated).md

```diff
@@ -0,0 +1,59 @@
+The Git Workflow
+==================
+
+## Overview
+
+By now, you should have a solid understanding of git commands, git architecture, and the difference between git and GitHub. It is time to put all of this together, by going through the git workflow.
+
+Imagine, you are at work and your manager wants you to make changes to a repository on GitHub called, *cookies*. Below, are step-by-step instructions on how to do just that.
+
+### Step 1
+
+Once you've located the repository on GitHub, clone it to your local machine. e.g.
+
+	$ git clone <remote repository URL>
+	
+This command will clone the repo from GitHub onto your local git repo.
+
+### Step 2
+
+Now that we have a copy of the *cookies* repo, we will create a branch called, *sidebar*. e.g.
+
+	$ git checkout -b sidebar
+	
+Branches are created in order to make changes to a file without merging those changes into **Master** branch. Changes should only be merged into the **Master** branch once they are final.
+
+Once you're satisfied with your changes, move the file from the **Working Directory** to the **Staging Index** e.g.
+
+	$ git add cake.html
```

Comment:
Your repo is called `cookies`, so make the file name `chocolate_chip.html` or `mint.html`. Other engineers would notice a naming mismatch like this, since `cake` is not a type of `cookie`.